src/wallet/rpcdump.cpp

```diff
@@ -1736,7 +1736,9 @@ RPCHelpMan listdescriptors()
     return RPCHelpMan{
         "listdescriptors",
         "\nList descriptors imported into a descriptor-enabled wallet.",
-        {},
+        {
+            {"private", RPCArg::Type::BOOL, /* default */ "false", "Show private descriptors."}
```

Comment:
I don't think this should be a numbered parameter.

Reply:
Are you saying it shouldn't be a parameter at all? Or it should be a "named-only" parameter? I can't see how to make it "named-only"

Reply:
Named-only parameters are provided in the positional API as an "options" Object.